packages/cursorless-engine/src/cursorlessEngine.ts

```diff
@@ -84,16 +90,34 @@ export async function createCursorlessEngine({
       )
     : new DisabledLanguageDefinitions();
 
+  const commandRunnerDecorators: CommandRunnerDecorator[] = [];
+
+  const addCommandRunnerDecorator = (decorator: CommandRunnerDecorator) => {
+    commandRunnerDecorators.push(decorator);
+  };
+
+  let tutorial: Tutorial;
+  if (tutorialContentProvider != null) {
+    const tutorialImpl = new TutorialImpl(
```

Comment:
Why not just a sign to `tutorial` directly here?

Reply:
because we need to know it is a `TutorialImpl` for the next two lines; a `Tutorial` doesn't have `dispose` or command runner decoration